androidApp/src/main/java/press/editor/EditorActivity.kt

```diff
@@ -94,6 +98,10 @@ class EditorActivity : ThemeAwareActivity() {
     private const val KEY_NOTE_ID = "press:new_note_id"
 
     private fun readNoteUuid(intent: Intent): Uuid {
+      if (intent.hasExtra(KEY_NOTE_ID).not()) {
+        // Adding uuid to intent extras if it already doesn't have it
+        intent.putExtra(KEY_NOTE_ID, uuid4().toString())
+      }
```

Comment:
Considering that `ShortcutReceiverActivity` opens this activity with a pre-set UUID, it seems inconsistent that this Activity is now also capable of handling missing UUIDs. Let's repurpose `ShortcutReceiverActivity` for opening `android.intent.action.SEND` as well?